src/runtime_src/core/edge/drm/zocl/zocl_cu_xgq.c

```diff
@@ -459,16 +442,16 @@ static inline void
 zcu_xgq_cmd_start_cuidx(struct zocl_cu_xgq *zcu_xgq, struct xgq_cmd_sq_hdr *cmd)
 {
 	struct kds_command *xcmd;
-	int mask_idx;
-	int bit_idx;
+	int mask_idx = 0;
+	int bit_idx = 0;
 
 #if 0
```

Comment:
Is this #if statement still needed.  If not, then maybe this code should be removed.